doc: Add NCP from library case to the known issues (#3164)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

KRKNWK-6408: ``diag`` command not supported
  The ``diag`` command is not yet supported by Thread in the |NCS|.

.. rst-class:: v1-4-0

> KRKNWK-7803: automatically generated libraries are missing otPlatLog for NCP

Reviewer: ```suggestion
KRKNWK-7803: Automatically generated libraries are missing otPlatLog for NCP
```

>   When building OpenThread libraries using different sample than the NCP, the ncp_base.cpp is not compiled with the otPlatLog funciton.

Reviewer: ```suggestion
  When building OpenThread libraries using a different sample than the :ref:`Thread NCP sample <ot_ncp_sample>`, the :file:`ncp_base.cpp` is not compiled with the :c:func:`otPlatLog` function.
```

>   It results in linking failure when building the NCP with use of these libraries.

Reviewer: ```suggestion
  This results in a linking failure when building the NCP with these libraries.
```

>   **Workaround:** Use NCP sample to create OpenThread libraries.

Reviewer: ```suggestion
  **Workaround:** Use the :ref:`Thread NCP sample <ot_ncp_sample>` to create OpenThread libraries.
```
